ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -12,6 +12,18 @@ interface {{ ethernet_interface }}
 {%         if ethernet_interfaces[ethernet_interface].speed is defined and ethernet_interfaces[ethernet_interface].speed is not none %}
    speed forced {{ ethernet_interfaces[ethernet_interface].speed }}
 {%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].priority_flow_control is defined and ethernet_interfaces[ethernet_interface].priority_flow_control == true %}
+   priority-flow-control on
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].priority_flow_control_priorities is defined and ethernet_interfaces[ethernet_interface].priority_flow_control_priorities is not none %}
+{%           for priorities in ethernet_interfaces[ethernet_interface].priority_flow_control_priorities %}
+{%             if ethernet_interfaces[ethernet_interface].priority_flow_control_priorities[loop.index0].enable_pfc == true  %}
+   priority-flow-control priority {{ ethernet_interfaces[ethernet_interface].priority_flow_control_priorities[loop.index0].priority }} no-drop
+{%             else %}
```

Comment:
By default, all priorities are set to drop and the EOS running configuration only requires no-drop to be explicitly configured. Please remove this logic as it is not required.